fastlane/lib/fastlane/actions/validate_play_store_json_key.rb

```diff
@@ -42,53 +42,49 @@ def self.example_code
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(
-            key: :json_key,
-            env_name: "SUPPLY_JSON_KEY",
-            short_option: "-j",
-            conflicting_options: [:json_key_data],
-            optional: true, # this shouldn't be optional but is until I find out how json_key OR json_key_data can be required
```

Comment:
Removed this comment because we already have `conflicting_options` in the place that is same as other actions...